handsontable/test/types/settings.types.ts

```diff
@@ -403,11 +403,33 @@ const allSettings: Required<Handsontable.GridSettings> = {
   afterContextMenuDefaultOptions: (predefinedItems) => {},
   afterContextMenuHide: (context) => {},
   afterContextMenuShow: (context) => {},
-  afterCopy: (data, coords) => {},
+  afterCopy: (clipboardData) => {
+    clipboardData.remove({ rows: [0], columns: [0] });
+    clipboardData.insertAtRow(0, ['A', 'B', 'C']);
+    clipboardData.insertAtColumn(0, ['A', 'B', 'C']);
+    clipboardData.setCellAt(0, 0, '');
+    clipboardData.getCellAt(0, 0, true);
+    clipboardData.getCellAt(0, 0);
+    clipboardData.getData();
+    clipboardData.getMetaInfo();
+    clipboardData.getSource();
+    clipboardData.getHotRanges();
+  },
   afterCopyLimit: (selectedRows, selectedColumnds, copyRowsLimit, copyColumnsLimit) => {},
   afterCreateCol: (index, amount, source) => {},
   afterCreateRow: (index, amount, source) => {},
-  afterCut: (data, coords) => {},
+  afterCut: (clipboardData) => {
+    clipboardData.remove({ rows: [0], columns: [0] });
```

Comment:
Are types correct? Because in the ClipboardData class, there is no `remove` method? 🤔

Reply:
It should be fixed.